src/rf/apps/core/migrations/0009_auto_20150908_1326.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```

Comment:
I meant to rename this but I was so excited that the build finally passed on Jenkins it slipped my mind. Will rename future migrations.